integration/package.mill

```diff
@@ -39,7 +39,14 @@ object `package` extends mill.Module {
       def mode: String = moduleSegments.parts.last
       def scalaVersion = Deps.scalaVersion
 
-      def forkEnv =
+      def forkEnv = {
+        def exportFromEnv(key: String) = {
+          sys.env.get(key) match {
```

Comment:
`sys.env` may return stale values. Use `Task.env` (or `TaskCtx.env`) instead.

```suggestion
          Task.env.get(key) match {
```

Reply:
Also, we still may suffer from stale cached values, since this isn't an input task.

Reply:
I'll push a fix